### PR TITLE
Present the correct signing data in the signing page

### PR DESCRIPTION
**Pull request by hamidra**
Currently the signing data content that is presented in the extension to be signed by the user removes the white space and new lines,
Screen Shot 2022-04-27 at 2 03 59 PM

This is not very user friendly and also might have security implications since the users are not presented with the actual data that are signing.

this changes make the signing data a scrollable multiline pre box to show the exact data in the signing box.
Screen Shot 2022-04-27 at 2 08 44 PM

### CODE REVIEW

File: packages/extension-ui/src/Popup/Signing/Bytes.tsx [hunk 1]
```diff
@@ -53,11 +54,24 @@ export default styled(Bytes)`
     text-overflow: ellipsis;
     vertical-align: middle;
     width: 100%;
+    padding: 5px;
```
Comment (jacogr) [inline packages/extension-ui/src/Popup/Signing/Bytes.tsx:57]: `0.125rem`? Just prefer to stick to one unit, can obviously adjust slightly.

File: packages/extension-ui/src/Popup/Signing/Bytes.tsx [hunk 2]
```diff
@@ -53,11 +54,24 @@ export default styled(Bytes)`
...
 
-    pre {
-      font-family: inherit;
-      font-size: 0.75rem;
-      margin: 0;
+    &.pre {
+      padding: 0px;
+      
+      div {
+        padding: 5px;
```
Comment (jacogr) [inline packages/extension-ui/src/Popup/Signing/Bytes.tsx:63]: As above, maybe `0.125rem`?

File: packages/extension-ui/src/components/Address.tsx [hunk 3]
```diff
@@ -306,7 +306,7 @@ function Address ({ actions, address, children, className, genesisHash, isExtern
 }
 
 export default styled(Address)(({ theme }: ThemeProps) => `
-  background: ${theme.accountBackground};
+  background: ${theme.boxBackground};
```
Comment (jacogr) [inline packages/extension-ui/src/components/Address.tsx:309]: Since it is only used in accounts, not sure the rename is needed? (It did refer to location)

**Review by jacogr — COMMENTED** [review]
Needs CI fixes. (`yarn lint` locally could auto-fix most/all, have not looked at the specifics)

File: packages/extension-ui/src/Popup/Signing/Bytes.tsx [hunk 4]
```diff
@@ -53,11 +55,24 @@ export default styled(Bytes)`
...
+        padding: 0.15rem;
+        font-family: inherit;
+        font-size: 0.75rem;
+        margin: 0;
+        white-space: pre;
+        overflow: auto;
+        max-height: calc(100vh - 480px);
+        min-height: ${theme.boxLineHeight};
+        border: 1px solid ${theme.boxBorderColor};
+        background: ${theme.boxBackground};
```
Comment (hamidra) [inline packages/extension-ui/src/Popup/Signing/Bytes.tsx:73]: used the boxBackground theme here.

File: packages/extension-ui/src/components/Address.tsx (hunk 3 above)
Comment (hamidra) [reply packages/extension-ui/src/components/Address.tsx:309]: I also used this for the signing data box background. Do you prefer to have different theme variables for account and box or just having a generic box background to be used for all type of these boxes makes sense?!

File: packages/extension-ui/src/Popup/Signing/Bytes.tsx (hunk 4 above)
Comment (jacogr) [reply packages/extension-ui/src/Popup/Signing/Bytes.tsx:73]: Ok, in that case, makes sense - not singular use.

**Review by jacogr — APPROVED** [review]
Thank you.

**Pull request merged**